csharp/ql/src/semmle/code/cil/Instructions.qll

```diff
@@ -167,7 +167,11 @@ module Opcodes {
   class Neg extends UnaryExpr, @cil_neg {
     override string getOpcodeName() { result = "neg" }
 
-    override NumericType getType() { result = getOperand(0).getType() }
+    override NumericType getType() {
+      result = getOperand().getType()
```

Comment:
missing `and not result instanceof Enum`?

Reply:
Not needed because `NumericType` and `Enum` are disjoint.